build/isc-bind9/files/named-template.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0"?>
+<!DOCTYPE service_bundle SYSTEM "/usr/share/lib/xml/dtd/service_bundle.dtd.1">
+<!--
+
+    This file and its contents are supplied under the terms of the
+    Common Development and Distribution License ("CDDL"), version 1.0.
+    You may only use this file in accordance with the terms of version
+    1.0 of the CDDL.
+
+    A full copy of the text of the CDDL should have accompanied this
+    source. A copy of the CDDL is also available via the Internet at
+    http://www.illumos.org/license/CDDL.
+
+    Copyright 2020 OmniOS Community Edition (OmniOSce) Association.
+
+-->
+
+<service_bundle type='manifest' name='named$(sVERSION)'>
+<service name="network/dns/named$(sVERSION)" type="service" version="1">
+
+<instance name='named$(sVERSION)' enabled='false'>
+
+	<dependency
+	    name='loopback'
+	    grouping='require_any'
+	    restart_on='error'
+	    type='service'>
+		<service_fmri value='svc:/network/loopback' />
+	</dependency>
+
+	<dependency
+	    name='network'
+	    grouping='optional_all'
+	    restart_on='error'
+	    type='service'>
+		<service_fmri value='svc:/milestone/network' />
+	</dependency>
+
+	<dependency
+	    name='filesystem_local'
+	    grouping='require_all'
+	    restart_on='none'
+	    type='service'>
+		<service_fmri value='svc:/system/filesystem/local:default' />
+	</dependency>
+
+	<dependent
+	    name='named$(sVERSION)_multi-user'
+	    grouping='optional_all'
+	    restart_on='none'>
+		<service_fmri value='svc:/milestone/multi-user' />
+	</dependent>
+
+	<exec_method
+	    type='method'
+	    name='start'
+	    exec='%{config/exec} -C %{config/file}'
+	    timeout_seconds='60'>
+            <method_context security_flags="aslr">
+		<method_credential user='$(USER)' group='$(USER)'
+                privileges="basic,net_privaddr,!priv_proc_info,!priv_proc_session" />
+	    </method_context>
+	</exec_method>
+
+        <exec_method
+            type="method"
+            name="stop"
+            exec=":kill"
+            timeout_seconds="60">
+        </exec_method>
```

Comment:
I know this doesn't match some of the other packages, but I'd consider moving this lot outside of the `<instance>`
That means that an administrator can create additional instances and they will properly inherit the dependencies.
The `config/` stuff remains per-instance.

Reply:
have kept as original, as it is now similar to nginx and apache, as discussed